Piccy/Piccy/View Controllers/Friends Page/FriendsViewController.m

```diff
@@ -372,12 +372,12 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
 }
 
 //Query for the friend requests tab
--(void) requestQuery:(NSString *)container {
+-(void) requestQuery:(NSString *)container withLimit:(int) limit {
```

Comment:
why you need to have the withLimit: ? seems like all the value you passed in is 10.